hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,92 @@
+defmodule HangmanTest do
```

Comment:
### Yea! Tests!  +6

I've already deducted for the fact that two of them had to be hacked to make them pass :)